### PR TITLE
Fix duplicate topic data in ConvertToLogPb()

### PR DESCRIPTION
**Pull request by dustinxie**

### CODE REVIEW

File: action/receipt_pre_aleutian.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import (
+	"github.com/golang/protobuf/proto"
+	"github.com/iotexproject/go-pkgs/hash"
+	"github.com/iotexproject/iotex-core/pkg/log"
+
+	"github.com/iotexproject/iotex-proto/golang/iotextypes"
+)
+
+// SerializePreAleutian converts a Log to protobuf's Log
+func (log *Log) ConvertToLogPbPreAleutian() *iotextypes.Log {
```
Comment (dustinxie) [inline action/receipt_pre_aleutian.go]: old buggy code before Aleutian height

File: Makefile [hunk 2]
```diff
@@ -79,7 +79,7 @@ fmt:
 
 .PHONY: lint
 lint:
-	go list ./... | grep -v /vendor/ | grep -v /explorer/idl/ | grep -v /api/idl/ | xargs $(GOLINT)
+	go list ./... | xargs $(GOLINT)
```
Comment (dustinxie) [inline Makefile]: these folders no longer exist in current codebase

File: action/protocol/execution/evm/evm_test.go [hunk 3]
```diff
@@ -24,38 +23,6 @@ import (
 	"github.com/iotexproject/iotex-core/testutil"
 )
 
-func TestLogReceipt(t *testing.T) {
```
Comment (dustinxie) [inline action/protocol/execution/evm/evm_test.go]: this test already included in receipt_test.go, so remove it

File: action/receipt.go [hunk 4]
```diff
@@ -101,7 +102,13 @@ func (log *Log) ConvertToLogPb() *iotextypes.Log {
 	l.ContractAddress = log.Address
 	l.Topics = [][]byte{}
 	for _, topic := range log.Topics {
-		l.Topics = append(l.Topics, topic[:])
+		if log.PreAleutian {
+			l.Topics = append(l.Topics, topic[:])
```
Comment (dustinxie) [inline action/receipt.go]: old buggy code for height < AleutianHeight

File: action/receipt.go [hunk 5]
```diff
@@ -101,7 +102,13 @@ func (log *Log) ConvertToLogPb() *iotextypes.Log {
 	l.ContractAddress = log.Address
 	l.Topics = [][]byte{}
 	for _, topic := range log.Topics {
-		l.Topics = append(l.Topics, topic[:])
+		if log.PreAleutian {
```
Comment (dustinxie) [inline action/receipt.go]: old buggy code

File: action/protocol/execution/evm/evm.go [hunk 6]
```diff
@@ -138,10 +145,10 @@ func ExecuteContract(
 	sm protocol.StateManager,
 	execution *action.Execution,
 	cm protocol.ChainManager,
-	pacificHeight uint64,
+	hc *HeightChange,
```
Comment (zjshen14) [inline action/protocol/execution/evm/evm.go]: Do not pass ptr. It's safer to pass struct directly

File: action/protocol/execution/protocol.go [hunk 7]
```diff
@@ -30,19 +30,19 @@ const (
...
-	addr          address.Address
-	pacificHeight uint64
+	cm   protocol.ChainManager
+	addr address.Address
+	hc   evm.HeightChange
 }
 
 // NewProtocol instantiates the protocol of exeuction
-func NewProtocol(cm protocol.ChainManager, pacificHeight uint64) *Protocol {
+func NewProtocol(cm protocol.ChainManager, pacificHeight, aleutianHeight uint64) *Protocol {
```
Comment (zjshen14) [inline action/protocol/execution/protocol.go]:
```suggestion
func NewProtocol(cm protocol.ChainManager, hc evm.HeightChange) *Protocol {
```

**Pull request merged**